automated install/basic-install.sh

```diff
@@ -2345,6 +2345,29 @@ migrate_dnsmasq_configs() {
 
     mv /etc/dnsmasq.d/0{1,2,4,5}-pihole*.conf "${V6_CONF_MIGRATION_DIR}/" 2>/dev/null || true
     mv /etc/dnsmasq.d/06-rfc6761.conf "${V6_CONF_MIGRATION_DIR}/" 2>/dev/null || true
+
+    # Finally, after everything is in place, we can create the new config file
+    # /etc/pihole/pihole.toml
+    # This file will be created with the default settings unless the user has
+    # changed settings via setupVars.conf or the other dnsmasq files moved above
+    # During migration, setupVars.conf is moved to /etc/pihole/migration_backup_v6
+    str="Migrating Pi-hole configuration to version 6"
+    printf "  %b %s..." "${INFO}" "${str}"
+    local FTLoutput FTLstatus
+    FTLoutput=$(pihole-FTL migrate v6)
```

Comment:
In the docker container - we move the conf files already - I suppose it is also worth running this new command too? 

I note the comments say the file will be created with defaults unless changes made in config files... will it also take into account environment variables?

Reply:
Re env vars: Yes, but only in new v6 style. The old v5 env vars are unknown to FTL.

Reply:
Re should it run in `docker`land?: Absolutely!

Reply:
Yeah, wouldn't expect v5 environment variables to be understood/migrated!